pkg/chunk/encoding/factory.go

```diff
@@ -15,7 +15,7 @@ type Config struct{}
 
 var (
 	// DefaultEncoding exported for use in unit tests elsewhere
-	DefaultEncoding             = DoubleDelta
+	DefaultEncoding             = Bigchunk
```

Comment:
This requires calling out in the changelog. Maybe that the storage size would increase slightly if there are lots of series with constant value